src/features/unified-share-modal/flowTypes.js

```diff
@@ -208,7 +208,7 @@ export type justificationReasonType = {
 };
 export type getJustificationReasonsResponseType = {
     classificationLabelId: string,
-    options: Array<justificationReasonType>,
+    options: Array<justificationReasonType> | null,
```

Comment:
I think this is equivalent
```suggestion
    options: ?Array<justificationReasonType>,
```